test/helper.js

```diff
@@ -102,6 +103,17 @@ class Helper {
       stdio: (global.Bare || global.process).argv.includes('--attach-boot-io') ? 'inherit' : 'ignore',
       cwd: this.platformDir
     })
+
+    sc.stdout.on('data', (data) => {
```

Comment:
just inherit the stdio (data can be chunked right? so this basically amounts to "run stdout:" being interspersed amongst the chunks)

Reply:
stdio set to 'inherit'